main.js

```diff
@@ -68,9 +68,13 @@ class ContentReviewLog {
             })).default;
             console.log(this._data);
         } catch (error) {
-            console.info(
-                'No cache.json file found, data will be created from scratch.'
-            );
+            if (error.code === 'ERR_MODULE_NOT_FOUND') {
+                console.info(
+                    'No cache.json file found, data will be created from scratch.'
+                );
+            } else {
+                throw new Error(error);
```

Comment:
You can just rethrow the same error.
```suggestion
                throw error;
```